src/plugins/data/public/search/session/session_service.ts

```diff
@@ -211,10 +242,52 @@ export class SessionService {
     this.refreshSearchSessionSavedObject();
   }
 
+  /**
+   * Continue previous search session
+   * Can be used to share a running search session between different apps, so they can reuse search cache
+   *
+   * This is different from {@link restore} as this doesn't try to reuse saved in elasticsearch search results when restoring
```

Comment:
I think this could be clearer if it stated where _it is_ reusing saved search results from (instead of where _it is not_ reusing search results from). E.g.: `This is different from {@link restore} as it reuses search results held in client memory instead of restoring search results from elasticsearch`. Might be misunderstanding the implementation, let me know if this is unclear 😅